spec/integration/kibana/knowledge_base_entry/manifest.spec.yml

```diff
@@ -0,0 +1,43 @@
+##
+## Describes the specification for a Kibana knowledge base entry's manifest.yml file
+##
+spec:
+  type: object
+  additionalProperties: false
+  properties:
+    name:
+      description: The name of the knowledge base entry
+      type: string
+    description:
+      description: A description of what the knowledge base entry provides
+      type: string
+    index:
+      type: object
+      additionalProperties: false
+      properties:
+        system:
+          description: Specify whether the index should be system-managed or not
+          type: boolean
+      required:
+        - system
+    retrieval:
+      type: object
+      additionalProperties: false
+      properties:
+        syntactic_fields:
+          description: List of fields that should be used for syntactic search during retrieval.
+          type: array
+          items:
+            type: string
+        semantic_fields:
```

Comment:
Where are these syntatic and semantic fields used? I wonder if we should add this to the descriptions.

Reply:
These are metadata that will be registered to the KB registry during the package installation, so that the KB knows which field to use to generate queries for retrieval.